src/app/routes/_question.router.js

```diff
@@ -27,19 +27,22 @@ export default (app, router, io, admin) => {
       const filterAnswered = req.query.filterAnswered === 'true';
       const page = req.query.page || 0;
       const query = filterAnswered
-        ? Question
-          .find({answered: { $eq: true }})
-          .limit(perPage)
-          .skip(perPage * page)
-        : Question
-          .find({answered: { $ne: true }})
-        ;
+          ? Question
+            .find({answered: { $eq: true }})
+            .limit(perPage)
+            .skip(perPage * page)
+          : Question
+            .find({answered: { $ne: true }})
+        // why is there no pagination?
```

Comment:
Because the number of unanswered question is always fairly small and there is no pagination in the Unanswered tab. Does this answer your question? Please remove these comments before merge.